<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDiluentDialog</class>
 <widget class="QDialog" name="AddDiluentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddDiluentDialog</string>
  </property>
  <widget class="QPushButton" name="pbAddDiluent">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Diluent</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbNotification">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>400</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Notification</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="pbProgress">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>400</width>
     <height>30</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
